scripts/mas_sso_config.sh

```diff
@@ -69,8 +69,6 @@ echo "$svcUserId"
 FINAL=$(curl -sk --data-raw '[{"id": '"$manageUser"',"name": "manage-users"},{"id": '"$manageRealm"',"name": "manage-realm"},{"id": '"$manageClients"',"name": "manage-clients"}]' --header "Content-Type: application/json" --header "Authorization: Bearer $TOKEN" "$KEYCLOAK_URL"/auth/admin/realms/rhoas/users/"$svcUserId"/role-mappings/clients/"$realmMgmtClientId")
 echo "$FINAL"
 
-
```

Comment:
This script is called mas_sso_config. There are other references in other parts. Is MAS SSO still used for something? should we be renaming elements?

Reply:
Is there some difference between mas sso and native keycloak?

Reply:
Yes, I think the real mas sso image has some extra capabilities that the native keycloak container doesnt have e.g enablement of  searchAttributes - to allow filtering service accounts (keycloak clients) by client's attributes. 
There could be other which I am missing @akoserwal can you think of others?

I believe we are using the native keycloak here and we do not have the mas-sso capabilities.

Reply:
> Is MAS SSO still used for something? should we be renaming elements?

@miguelsorianod I've actually remembered that some time ago I created this https://issues.redhat.com/browse/MGDSTRM-3890 to rename the sso related flags (to be fair, everything) so that they are generic enough. I think the work could be progressed there?